docs/quickstarts/registry.rst

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
This file doesn't belong in the `quickstarts` directory. Please put it in the top-level `docs` directory, and add a reference to it in the table of contents in the `docs/index.rst` file.